Dockerfile_cli

```diff
@@ -0,0 +1,17 @@
+# Dockerfile for building the cli of download with
+# ability of multifolder downloading
+
+
+
+ARG BASE_IMAGE=nansencenter/geospaas:2.0.0-slim
+FROM ${BASE_IMAGE} AS base
+RUN pip install --no-cache-dir django-celery-results==1.2 redis graypy
```

Comment:
Is that needed? setup.py requires only 'django-geo-spaas', 'paramiko', 'scp' which are probably in the geospaas image already.
```suggestion
```

Reply:
`django-celery-results` is needed because it is imported in the source code. the last two ones are imported in the test file so they are needed for accomplishing the tests.